.eslintrc.json

```diff
@@ -17,6 +19,13 @@
   },
   "rules": {
     "prettier/prettier": "error",
-    "react/jsx-uses-vars": "error"
+    "react/jsx-uses-vars": "error",
+    "jsx-a11y/click-events-have-key-events": "warn",
```

Comment:
We currently have 13 jsx-ally errors. It's too big for one single PR to cover all of them so I set the related rules to raise warning instead. That way they won't break our tests and we can tackle them in several PRs.

@Pomax if this approach sounds good, I will file a PR to make sure we remove these temporary jsx-ally config from this file once we fix all jsx-ally issues.
<img width="1415" alt="Screen Shot 2020-03-11 at 6 20 14 PM" src="https://user-images.githubusercontent.com/2896608/76541814-5fbb8c80-6441-11ea-99db-a036ab3fe2b7.png">